GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -92,6 +93,7 @@ public string ServicePipeName
         }
 
         public TextWriter Output { get; set; }
+        public TextWriter ErrorOutput { get; set; }

```

Comment:
Should this only be used for machine readable output?  Or in the future should new error messages be written to `ErrorOutput`?

Reply:
I really don't think we want to put that restriction on this.  We haven't been writing to stderr and it isn't displayed by default in the console.  If we want to really create a machine readable CLI then we should do something else but not sure how much work we want to put into it.

Reply:
> it isn't displayed by default in the console

I did not realize this thanks!

Sounds like this change is good as it is now.

Reply:
I will still add a comment about how we are using this right now in case something else wants to use it.

Reply:
Sorry it's not Console doesn't show stderr.  It does but I wasn't seeing the stderr stream from the DehydrateVerb in the sparse console output without using `2>&1`, so maybe there is something more I need to do here.

Reply:
Weird it is working the way you would expect.  Not sure what was going on when I didn't see the folder list in the console, maybe I didn't have everything connected yet.